tests/unit/worker/task/test_feature_job_setting_analysis.py

```diff
@@ -71,6 +72,19 @@ def use_mock_event_dataset(self, mock_event_dataset):
         _ = mock_event_dataset
         yield
 
+    @staticmethod
+    def clean_html(html):
+        """
+        Strip img tag from html
+        """
+        soup = bs4.BeautifulSoup(html, "html.parser")
+        for tag in soup.find_all("img"):
+            # strip image tag with base64 encoded image to avoid issue where different OS may
+            # generate different base64 encoded image (could be due to different font rendering)
+            if tag["src"].startswith("data:image/png;base64"):
+                tag.decompose()
+        return str(soup)
```

Comment:
@kchua78 FYI, I update the test here to exclude the image tags from the html as the generated images are slightly different between Linux & MacOS.